raster/src/main/scala/geotrellis/raster/rasterize/Rasterizer.scala

```diff
@@ -205,6 +221,29 @@ object Rasterizer {
     * LineString.  The iteration happens in the direction from the
     * first point to the last point.
     */
+  def foreachCellByLineString(
+    line: Line,
+    re: RasterExtent,
+    options: Options
+  )(f: (Int, Int) => Unit) {
+    val cells = (for(coord <- line.jtsGeom.getCoordinates()) yield {
+      (re.mapXToGrid(coord.x), re.mapYToGrid(coord.y))
+    }).toList
```

Comment:
Unless I misunderstand what you are saying, [this is a copy of the existing code](https://github.com/locationtech/geotrellis/blob/master/raster/src/main/scala/geotrellis/raster/rasterize/Rasterizer.scala#L208-L219).  Does it need to be changed too?